aml-development/frontend
# feat(edit profile): updates to edit profile page

AMLOS-554

**Description**
As a user, I want to view my user profile, and configure my settings

**Acceptance Criteria**
- User profile window is available
- Details about the user is in the profile window (name/org/work role)
- Settings are available to change – email settings, notification settings, theme settings
- User's listings are available
- Use can customize pieces of their profile

**How to Test**
Pull `edit_profile_updates` from `frontend`
If https://github.com/aml-development/ozp-backend/pull/433 is not merged, pull `work_role_metadata` from `ozp-backend`

User can view their profile page by clicking the icon on the nav menu
- Profile page has username, org, and work roles (affiliation is TBD)
- User can see their listings in the My Listings section

User can edit their profile page by clicking the pencil icon below the nav menu
- Name and Org show, but are grayed out (uneditable)
- Work Role(s) can be edited and saved successfully
- Toggled preferences can be switched on/off and saved successfully
- Site theme can be changed and saved successfully
- The remaining items (affiliation, categories, and tags) are TBD
